docs/api_guide/tutorial_pynapple_core.py

```diff
@@ -126,21 +132,83 @@
 print(count)
 
 # %%
-# One advantage of grouping time series is that metainformation can be appended directly on an element-wise basis. In this case, we add labels to each Ts object when instantiating the group and after. We can then use this label to split the group. See the [TsGroup](https://peyrachelab.github.io/pynapple/core.ts_group/) documentation for a complete methodology for splitting TsGroup objects.
-
+# One advantage of grouping time series is that metainformation can be added directly on an element-wise basis. In this case, we add labels to each Ts object when instantiating the group and after. We can then use this label to split the group. See the [TsGroup](https://peyrachelab.github.io/pynapple/core.ts_group/) documentation for a complete methodology for splitting TsGroup objects.
+#
+# First we create a pandas Series for the label.
 
 label1 = pd.Series(index=list(my_ts.keys()), data=[0, 1, 0])
 
-tsgroup = nap.TsGroup(my_ts, time_units="s", label1=label1)
-tsgroup.set_info(label2=np.array(["a", "a", "b"]))
+print(label1)
 
-print(tsgroup, "\n")
+# %%
+# We can pass `label1` at the initialization step.
+
+tsgroup = nap.TsGroup(my_ts, time_units="s", my_label1=label1)
+
+print(tsgroup)
+
+# %%
+# Notice how the label has been added as one column when printing `tsgroup`.
+#
+# We can also add a label for each items in 2 different ways after initializing the `TsGroup` object.
+# First with `set_info` :
+tsgroup.set_info(my_label2=np.array(["a", "a", "b"])) 
+
+print(tsgroup)
+
+# %%
+# Notice that you can pass directly a numpy array as long as it is the same size as the `TsGroup`.
+# 
+# We can also add new metadata by passing it as an item of the dictionnary.
```

Comment:
Maybe add that the key must be a string (otherwise if it is an integer, the getitem would be conflicting with the neuron's keys).

Dictionary has a single "n"